vulnerabilities/pipelines/risk_package.py

```diff
@@ -0,0 +1,52 @@
+from aboutcode.pipeline import LoopProgress
+
+from vulnerabilities.models import Package
+from vulnerabilities.pipelines import VulnerableCodePipeline
+from vulnerabilities.risk import calculate_pkg_risk
```

Comment:
Try to be consistent with the wording, "compute"/"calculate", "package/pkg", ...
Also, `risk_package` as the pipeline module name does not follow the conventions.
`compute_package_risk` would be much better, as defined in `pipeline_id`.